lib/full_text_search/issue_synchronizable.rb

```diff
@@ -0,0 +1,29 @@
+module FullTextSearch
+  module IssueSynchronizable
+    extend ActiveSupport::Concern
+
+    included do
+      after_commit :queue_sync_on_commit
+      after_destroy :queue_sync_on_destroy
+    end
+
+    private
+
+    def queue_sync(action, options = {})
```

Comment:
Do we need `options` here?

Reply:
fix: [4f556ff](https://github.com/clear-code/redmine_full_text_search/pull/167/commits/4f556ffa56e5f9d0fbff3b84fde92d57ec12aac0) We don't need it.